### ISSUE TITLE
Low data regime in ratio and not fixed count as reported in the paper

### ISSUE THREAD

**Issue by ShambAI** [Navidfoumani/Series2Vec#11]
Hi,

Thank you for open-sourcing your code. 

I am trying to reproduce the results on the low-label regime experiments. I noticed the split_dataset function under your dataloader is not for fixed count as reported in the paper but rather ratio (stratified ratio/percentage). Is there something I am missing?

**Comment by Navidfoumani** [comment]
Hi,

You’re right that the split_dataset function is based on ratios and is used for the main experiments.
For the low-label regime experiments, I simply randomly selected the desired number of samples from the training set. Note that some datasets don’t have 500 samples per class, so to keep experiments consistent across datasets, I used all available samples for that class and repeated them until reaching 500.

Hope this clarifies!

**Comment by ShambAI** [comment]
Thank you for your response. 

I am unable to reproduce the results you got for the low label regime. Notably, for the 1, 5, and 10 samples per class.
